onnxruntime/core/framework/cloud_invoker.cc

```diff
@@ -196,10 +187,8 @@ onnxruntime::Status OpenAIInvoker::Send(const CloudEndPointConfig& run_options,
 
   auto* output_string = output_tensor->MutableData<std::string>();
   *output_string = string_buffer.ss_.str();
-
-  ort_outputs.resize(1);
   auto tensor_type = DataTypeImpl::GetType<Tensor>();
```

Comment:
```
  auto tensor_type = DataTypeImpl::GetType<Tensor>();
``` 

What is this for?

Reply:
Like the code indicate - it is MLDATATYPE refers to a tensor type, we use that for creating an output value of tensor.

Reply:
How about 
```
  ort_outputs.emplace_back(std::move(output_tensor));
```
Does it work?